src/esportsbot/cogs/MusicCog.py

```diff
@@ -0,0 +1,1117 @@
+                await message.delete()
+                await self.__send_timed_message(message.channel, send, timer=10)
+                return
+
+        # Check if the loops for marked and active channels are running.
+        self.__check_loops_alive()
+
+        # Splits multiline messages into a list. Single line messages return a list of [message]
+        split_message = message.content.split("\n")
+        split_message = list(filter(lambda k: k.replace(' ', '') != '', split_message))
+        partial_success = True
+
+        # Add each line of the message to the queue
+        st = time.time()
+        for line in split_message:
+            partial_success = await self.process_song_request(message, line) or partial_success
+        print('Queue time:' + str(time.time() - st))
+        # If any of the songs were successful, ensure that the channel is no long marked as inactive.
+        if partial_success:
+            if message.guild.id in self._marked_channels:
+                # Remove the channel from marked channels as it is no longer inactive
+                self._marked_channels.pop(message.guild.id)
+
+        await message.delete()
+
+    async def process_song_request(self, message, request):
+        """
+        Process a song request.
+        :param message: The instance of the discord message that sent the request.
+        :param request: The actual message request.
+        :return The success value of if the song was added to the queue or not.
+        """
+
+        message_type = self.__determine_message_type(request)
+        if message_type == 1:
+            return await self.__add_song_to_queue(message.guild.id, request)
+        elif message_type == 0:
+            individual_links = self.__find_playlist_songs(request)
+            return await self.__add_song_to_queue(message.guild.id, individual_links)
+        elif message_type == 2:
+            queried_song = self.__find_query(request)
+            return await self.__add_song_to_queue(message.guild.id, queried_song)
+
+    async def __update_channel_messages(self, guild_id):
+        """
+        Update the queue and preview messages in the music channel.
+        :param guild_id: The guild id of the guild to be updated.
+        """
+
+        guild_db_data = db_gateway().get('music_channels', params={'guild_id': guild_id})[0]
+
+        # Get the ids of the queue and preview messages
+        queue_message_id = guild_db_data.get('queue_message_id')
+        preview_message_id = guild_db_data.get('preview_message_id')
+
+        # Create the updated messages
+        queue_message = self.__make_updated_queue_message(guild_id)
+        preview_message = self.__make_update_preview_message(guild_id)
+
+        music_channel_id = guild_db_data.get('channel_id')
+        # Get the music channel id as a discord.TextChannel object
+        music_channel_instance = await self._bot.fetch_channel(music_channel_id)
```

Comment:
it'll probably be in cache, in which case this is quite a bit faster:
```
        music_channel_instance = self._bot.get_channel(music_channel_id)
        if music_channel_instance is None:
            music_channel_instance = await self._bot.fetch_channel(music_channel_id)
```

We even know the guild ID here, so we can do this:
```suggestion
        music_channel_instance = self._bot.get_guild(guild_id).get_channel(music_channel_id)
        if music_channel_instance is None:
            music_channel_instance = await self._bot.fetch_channel(music_channel_id)
```